www/src/pages/en/deployment/vercel.md

```diff
@@ -6,19 +6,47 @@ layout: ../../../layouts/docs.astro
 
 We recommend deploying your app to [Vercel](https://vercel.com/?utm_source=t3-oss&utm_campaign=oss). It makes it super easy to deploy Next.js apps.
 
-1. Push your code to a GitHub repository.
-2. Go to [Vercel](https://vercel.com/?utm_source=t3-oss&utm_campaign=oss) and sign up with GitHub.
-3. Click on **Add New Project**.
+## Project Configuration
 
-   ![New project on Vercel](/images/vercel-new-project.webp)
+Vercel will likely configure your build command and publish directory automatically. However, you can also specify this information along with other configuration by creating a file called [`vercel.json`](https://vercel.com/docs/project-configuration) and including the following commands:
 
-4. Import the GitHub repository with your project.
+```json
+{
+  "buildCommand": "pnpm build",
+  "outputDirectory": "dist",
+  "devCommand": "pnpm dev",
+  "installCommand": "pnpm install"
+}
+```
 
-   ![Import repository](/images/vercel-import-project.webp)
+## Using the Vercel Dashboard
 
-5. Add your environment variables.
+1. After pushing your code to a GitHub repository, sign up for [Vercel](https://vercel.com/?utm_source=t3-oss&utm_campaign=oss) with GitHub and click on **Add New Project**.
 
-   ![Add environment variables](/images/vercel-env-vars.webp)
+![New project on Vercel](/images/vercel-new-project.webp)
 
-6. Click **Deploy**.
-7. Now whenever you push a change to your repository, Vercel will automatically redeploy your website!
+2. Import the GitHub repository with your project.
+
+![Import repository](/images/vercel-import-project.webp)
+
+3. Add your environment variables.
+
+![Add environment variables](/images/vercel-env-vars.webp)
+
+4. Click **Deploy**. Now whenever you push a change to your repository, Vercel will automatically redeploy your website!
+
+## Using the Vercel CLI
+
+To deploy from the command line you can either:
+
+- [Install the Vercel CLI globally](https://vercel.com/docs/cli#installing-vercel-cli) with `pnpm i -g vercel`.
+
+- Add the CLI as a dependency to your project with `pnpm add -D vercel` and run commands with `pnpm vercel`.
```

Comment:
Is this step necessary? Can you not just use the global cli?

Reply:
It's a philosophical debate. Some people prefer avoiding global installs cause then everyone gets the same version and also you can just clone down the repo and run the command without needing to worry about installing anything onto your machine which can be a pain when there's a bunch of different ways to install a CLI. In fact Vercel's command for installing globally with `pnpm` is actually incorrect right now and says `pnpm i -g vercel`.